whylog/tests/tests_front/test_unit_whylog_front.py

```diff
@@ -0,0 +1,64 @@
+from mock import patch
+from unittest import TestCase
+from unittest.mock import MagicMock
+from whylog.front import WhylogFront
+from whylog.front.front_input import FrontInput
+from whylog.front.resource_location import LocallyAccessibleLogOpener
+
+
+def get_test_front_input():
+    log_opener = LocallyAccessibleLogOpener(path = '')
+    return FrontInput(
+        offset=1,
```

Comment:
for such data offset will be `0` (unless the first line of the file is empty, but I doubt this test is about that)